account_asset_comment/models/models.py

```diff
@@ -0,0 +1,11 @@
+# Copyright NuoBiT Solutions, S.L. (<https://www.nuobit.com>)
+# Eric Antones <eantones@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class AccountAsset(models.Model):
+    _inherit = "account.asset"
+
+    note = fields.Text(string="Comment")
```

Comment:
@MiquelRForgeFlow  It's not necessary to rename the field, the default "Note" name is fine, you can remove this.